UGentBiomath/COVID19-Model
# Full pandemic calibration

* [x] I have checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change
* [x] I made sure all new and existing data sets have a description in the `data/README.md` file

Describe your fixes/additions/changes:
- Take out the Arenas terms in the spatial model (because differentiation between different regions is already done with the introduction of the three distinct beta factors)
- Change the boundaries of the PSO and MCMC based on national results
- Extend the `initial_states` function such that it allows for three types of age stratification
- Update latest Sciensano data from the remote desktop
- Extend the load function of spatially explicit Sciensano data to allow for loading public data
- Use public data for the FULL calibration, use private data for the R0 calibration
- Add results. Note: PSO results seem fair, but 100 MCMC iterations appear to diverge from this reasonable first-result.

Current step: merge these changes to the main branch, synchronise HPC, add data to HPC, and perform a long calibration on the full data. Hopefully this will converge.
